types/react-native-dotenv/index.d.ts

```diff
@@ -4,9 +4,5 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
-interface Env {
-    [name: string]: string;
-}
-
-declare const env: Env;
-export = env;
+declare const _a: any;
```

Comment:
I'm not quite sure what this change is intending to solve. Could you add a test case to give an example of the problem and how this fixes it?

Reply:
In  #34600, it was recommended to import the library `import env from "react-native-dotenv"`.
However if you try this it throw an error, as per this issue  zetachang/react-native-dotenv#5. 
So thats what I'm trying to resolve.